menuinst/platforms/win.py

```diff
@@ -447,7 +447,9 @@ def _unregister_file_extensions_cmd(self):
             # TODO: Do we need to clean up the `assoc` mappings too?
 
     @staticmethod
-    def _cmd_assoc(extension, associate_to=None, query=False, remove=False) -> CompletedProcess:
+    def _cmd_assoc(
+        extension, associate_to: str | None = None, query: bool = False, remove: bool = False
```

Comment:
```suggestion
        extension: str, associate_to: str | None = None, query: bool = False, remove: bool = False
```

Based on the code in this function, `extension` should be `str`.